pepper-apis/studybuilder-cli/src/main/java/org/broadinstitute/ddp/studybuilder/task/SingularAboutHealthySelectLogic.java

```diff
@@ -0,0 +1,220 @@
+package org.broadinstitute.ddp.studybuilder.task;
+
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.Optional;
+
+import com.typesafe.config.Config;
+import com.typesafe.config.ConfigException;
+import com.typesafe.config.ConfigFactory;
+import lombok.extern.slf4j.Slf4j;
+import org.apache.commons.lang3.StringUtils;
+import org.broadinstitute.ddp.db.DBUtils;
+import org.broadinstitute.ddp.db.dao.FormActivityDao;
+import org.broadinstitute.ddp.db.dao.JdbiActivity;
+import org.broadinstitute.ddp.db.dao.JdbiActivityVersion;
+import org.broadinstitute.ddp.db.dao.JdbiPicklistOption;
+import org.broadinstitute.ddp.db.dao.JdbiUmbrellaStudy;
+import org.broadinstitute.ddp.db.dto.StudyDto;
+import org.broadinstitute.ddp.exception.DDPException;
+import org.broadinstitute.ddp.model.activity.definition.FormActivityDef;
+import org.broadinstitute.ddp.model.activity.definition.question.PicklistQuestionDef;
+import org.broadinstitute.ddp.model.activity.types.ActivityType;
+import org.broadinstitute.ddp.model.activity.types.QuestionType;
+import org.jdbi.v3.core.Handle;
+
+/**
+ * Task for updating the IF_YOU_HAVE_HAD_ARRHYTHMIA_COMPLICATIONS (PATIENT_SURVEY)
+ * and IF_YOU_HAVE_HAD_ARRHYTHMIA (ABOUT_HEALTHY) questions to mark the DO_NOT_REQUIRE
+ * and DID_NOT_REQUIRED options (respectively) as exclusive.
+ * 
+ * <p>See https://broadinstitute.atlassian.net/browse/DDP-8576
+ */
+@Slf4j
+public class SingularAboutHealthySelectLogic implements CustomTask {
```

Comment:
Suggestions for names would be welcomed here- this name is somewhat misleading as this is altering both the `ABOUT_HEALTHY` and `PATIENT_SURVEY` activities.

Reply:
How about SingularMakeNoMedicalTreatmentOptionExclusive? (Though I am going to drink very large beer in celebration when we get rid of the patch thing!!!)

Reply:
Seconded 💯.

That name works for me.